release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -1653,7 +1655,7 @@ This update corrects the accepted label specification so that the plug-in import
 
 * Previously, RAM validation for {rh-openstack-first} checked for values using a wrong unit, and as a result the validation accepted flavors that did not meet minimum RAM requirements. With this fix, RAM validation now rejects flavors with insufficient RAM. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2009699[*BZ#2009699*])
 
-* Previously, {product-title} master nodes were missing Ingress security group rules when they were schedulable and deployed on {rh-openstack}. As a result, {product-title} deployments on {rh-openstack} failed for compact clusters with no dedicated workers. This fix adds Ingress security group rules on {rh-openstack-first} when masters are schedulable. Now, you can deploy compact three-node clusters on {rh-openstack}. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1955544[*BZ#1955544*])
+* Previously, {product-title} control plane nodes were missing Ingress security group rules when they were schedulable and deployed on {rh-openstack}. As a result, {product-title} deployments on {rh-openstack} failed for compact clusters with no dedicated workers. This fix adds Ingress security group rules on {rh-openstack-first} when control plane nodes are schedulable. Now, you can deploy compact three-node clusters on {rh-openstack}. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1955544[*BZ#1955544*])
```

Comment:
Since this is a new entry, do we want to call out {rh-openstack-first}? Followed by {rh-openstack} with the second entry.